### PR TITLE
Mods for 5.x

### PR DESCRIPTION
**Pull request by jamesbeedy**
* Conditionals were added to the elasticsearch.yaml template
to facilitate deprecated configs in 5.x that are needed in > 5.x,
and for configs that cannot exist with no value in 5.x that could
pre 5.x.

* Turn off verify certs for 5.x.

* Modified install-elasticsearch.yml to facilitate not verifying certs when adding the apt key.

* Add single deploy test for ES 5.x.

* Update readme

* Add series to metadata.yaml

### CODE REVIEW

File: tasks/install-elasticsearch.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     - install
     - upgrade-charm
     - config-changed
-  apt_key: url={{ apt_key_url }} state=present id={{gpg_key_id}}
+  apt_key: url={{ apt_key_url }} state=present id={{gpg_key_id}} validate_certs=no
```
Comment (marcoceppi) [inline tasks/install-elasticsearch.yml:6]: What's the reason for this?

Comment (jamesbeedy) [reply tasks/install-elasticsearch.yml:6]: @marcoceppi http://paste.ubuntu.com/24470336/

Comment (marcoceppi) [reply tasks/install-elasticsearch.yml:6]: I find it odd that elastic search repo doesn't have valid certs...

Comment (jamesbeedy) [reply tasks/install-elasticsearch.yml:6]: @marcoceppi you can simply check it yourself in the browser https://artifacts.elastic.co/GPG-KEY-elasticsearch

The apt-key-url changed from pre 5. releases, you can see its not even wrapped in anything here http://packages.elasticsearch.org/GPG-KEY-elasticsearch

**Comment by jamesbeedy** [comment]
Picking up these changes in add_xenial_support branch.

**Pull request closed without merging**